autospec/avxjudge.py

```diff
@@ -0,0 +1,512 @@
+#!/usr/bin/python3
+import subprocess
+import sys
+import re
+import argparse
+import os
+
+# MMX and SSE2 instructions
+sse_instructions_xmm = list()
```

Comment:
This entire file should be re-written into idiomatic python. I'll put suggestions in a few places below but I won't comment on everything.

First thing, this can definitely be refactored to not need so many globals.